src/py/crankshaft/crankshaft/query_runner.py

```diff
@@ -0,0 +1,43 @@
+"""class for fetching data"""
+import plpy
+
+
+class QueryRunner:
+    def get_markov(self, query):
```

Comment:
Why do you need to provide the query? I would try to encapsulate the query generation within this class, making this method to handle the required params (num_ngbrs,...).